nginx.conf

```diff
@@ -322,7 +326,7 @@ echo "Docker configured with HTTPS_PROXY=$scheme://$http_host/"
             proxy_cache $cache;
             # But we store the result with the cache key of the original request URI
             # so that future clients don't need to follow the redirect too
-            proxy_cache_key $original_uri$slice_range;
+            proxy_cache_key $orig_cache_key$slice_range;
```

Comment:
is $orig_cache_key even populated in @handle_redirects path?

I think this should be (and drop orig_cache_key)
```
            proxy_cache_key $request_method$original_uri$slice_range;
```